src/main/java/bio/terra/janitor/app/configuration/PrimaryConfiguration.java

```diff
@@ -30,6 +30,12 @@
   /** How many flights to process for completion at a time. */
   private int flightCompletionLimit = 1000;
 
+  /** How often to query for flights that have completed fatally. */
+  private Duration fatalFlightCompletionPeriod = Duration.ofMinutes(1);
```

Comment:
Do we need different configs for the differrent state? They looks the same now. Is there any known reason we may have different period and limit?

Reply:
Yeah, they could process at different speeds, so we might want to give them shorter or longer periods. We might also see different rates for the limits.
Started them the same since I don't know which way they'll go.